src/main/java/io/openliberty/tools/common/plugins/util/PrepareFeatureUtil.java

```diff
@@ -251,8 +251,10 @@ public File generateJson(String targetJsonFile, Map<File, String> esaFileMap) th
 				json = (File) mapBasedInstallKernel.get("generate.json");
 				
 				if (mapBasedInstallKernel.get("action.error.message") != null) {
+					debug("generateJson action.error.message: "
+							+ mapBasedInstallKernel.get("action.error.message"));
 					debug("generateJson action.exception.stacktrace: "
-							+ mapBasedInstallKernel.get("action.error.stacktrace"));
+							+ mapBasedInstallKernel.get("action.exception.stacktrace"));
```

Comment:
We have to work with various different versions of Liberty. If this key changed in a recent version, seems like this code would need to handle both keys?

Reply:
Sorry I should have mentioned this is actually a bug. The key has been 'action.exception.stacktrace' for a long period of time. I will check for any additional references to this key.